torch/distributed/utils.py

```diff
@@ -0,0 +1,31 @@
+import torch.distributed as dist
+
+def _verify_param_shape_across_processes(process_group, tensors, logger=None):
+    return dist._verify_params_across_processes(process_group, tensors, logger)
+
+def _sync_params_and_buffers(
```

Comment:
nit: Do we want to add some type annotations?

Reply:
I think we should probably decide whether we want to invest in typing across distributed codebase and take that on as a BE task?